<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventLayerDialogBase</class>
 <widget class="QDialog" name="EventLayerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>459</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add event layer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="mPointRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Point</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="mLineRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mLineLayeLabel">
     <property name="text">
      <string>Line layer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="mLineLayerComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mLineFieldLabel">
     <property name="text">
      <string>Line field</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="mLineFieldComboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mEventLayerLabel">
     <property name="text">
      <string>Event layer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="mEventTableComboBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mEventFieldLabel">
     <property name="text">
      <string>Event field</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="mEventFieldComboBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mFromFieldLabel">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="mFromFieldComboBox"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mToFieldLabel">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="mToFieldComboBox"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="mOffsetGroupBox">
     <property name="title">
      <string>Offset</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="mOffsetFieldLabel">
        <property name="text">
         <string>Offset field</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mOffsetFieldComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mOffsetScaleLabel">
        <property name="text">
         <string>Offset scale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="mOffsetScaleSpinBox">
        <property name="minimum">
         <double>-9999999.990000000223517</double>
        </property>
        <property name="maximum">
         <double>99999999.989999994635582</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="mOutputGroupBox">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mOutputNameLabel">
        <property name="text">
         <string>Output name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mOutputLayerNameEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="mForceSingleTypeCheckBox">
        <property name="text">
         <string>Force single type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="btnRefresh">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
